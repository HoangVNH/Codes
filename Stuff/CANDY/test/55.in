10
74
XDVXDDVVDVXVXXVVVDXXXVDXXVVDXVXXDVVDXXVDVVDDXVVVDXVDDDDDDDVDVVVXVVVXVDVXDV
86
XDDVDXVXVXXVDDVXDXXVDVDVXXXVVDXDXXDXXVDXDDXXVVDXXXDXVDXVDDDVXDVVXVXXXVXVVVVVXXVVVVDXVD
98
DVVVVVVDVDXDDVXVXXDDDXVDXVDXVXDXXDDDDDDVDXDXVVDXDVDXDXXVVDVXVXDXVDDVXDDVDDDXVXVXVDXXXDVVDXVVXDXXVX
85
DXVVVXXXXXXXDVXXDXVVXXDXVXXDVDDVXDXVVVDVVXDDXXVVVDVDDXDXDVXXXVDDDVDDVVVXDVVVDDXVDXXXX
80
XDXXVVVDDVVDXVXXVDDXVDDDDVVDXVVDDDVXXDVXDDVVXDXVDXDXDVXVXDVVXVVDDDDVXVXDVDVVDDXX
81
DDDXDVXDXDDDXXDVDDXDDXVDVDVXVDVDVVXDXXDDXDVVXXDVDXDVDXDVVXVDVXDVVVXVVVXVDXXXDDXDV
74
VDVDDXVVVDVDDDDXVVVXDDXDVDXVVXXVVXVXXXVXXDVDDVXXVDVDDXDDDVVDVDVVXVVXXDXVDD
78
XXVVDXXVVXDXVDVDXVXDXVVXDXDXXXVDXDDDVVXXDXXVVDXXVVDDDVVXVDVXVDVVXDXXDDXVVXVVDV
88
XXXVXXVDVVXXDXVXXDVDDDXXDXXXXVVVDDVVDXDVDVVVVXDDXDXVXVXDVDXDVDDXVXVDVXXXXXXXVXDXVDXVVVVV
56
VXVDXDDVDVDDXDVVVXVDDXDXDDXXVDDXDXDVDDXXVDXVDDDVVVXVVXVV
