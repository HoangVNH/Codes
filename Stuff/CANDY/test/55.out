73
85
97
84
78
80
72
76
87
55
